manabi_django/manabi_migrations/models.py

```diff
@@ -2,7 +2,7 @@
 
 
 class Lock(models.Model):
-    token = models.CharField(max_length=128, primary_key=True)
+    token = models.CharField(max_length=250, primary_key=True)
```

Comment:
Wouldn't it make sense to use 255 as that is standard practice?

Reply:
makes sense, updated